### PR TITLE
Initial doc/readme

### PR DESCRIPTION
**Pull request by gabenitez**
commiting initial readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
-discordBot
+# Descripción 
+
+An experimental bot (initially for discord) that allows us to manage a reward system through coins and items that can be purchased
+
+It allows:
+- Manage balances in wallets
+- Transfer balances between members' wallets
```
Comment (Sebas37) [inline README.md]: Bad typing
```suggestion
- Transfer balances between member's wallets
```

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+
+## Admin only commands
+```
+/emmit [amout]
+		Transfers [amount] monedas a un usuario
+
+/createItem [itemName] [ItemDescription] [ItemAmount] [ItemPrice]
+		Create an item in the servers store
+
+```
```
Comment (Sebas37) [inline README.md]: The commands are @RewardsBot <command> <...params>

**Review by Sebas37 — CHANGES_REQUESTED** [review]
Refused, read my comments.

File: README.md (hunk 1 above)
Comment (gabenitez) [reply README.md]:
```suggestion
- Transfer balances between member's wallets
```

File: README.md (hunk 2 above)
Comment (gabenitez) [reply README.md]: ok

**Review by Sebas37 — APPROVED** [review]

**Pull request merged**